packages/scripts/README.md

```diff
@@ -350,8 +350,8 @@ _Example:_
 
 This is how you execute the script with presented setup:
 
--   `npm start` - starts the build for development.
--   `npm start:hot` - starts the build for development with "Fast Refresh". The page will automatically reload if you make changes to the code.
+-   `npm run start` - starts the build for development.
```

Comment:
Actually, `npm start` is correct. It's a shorthand as documented in https://docs.npmjs.com/cli/v8/commands/npm-start.
```suggestion
-   `npm start` - starts the build for development.
```